yarnitor/background/worker.py

```diff
@@ -34,260 +38,390 @@
 THREADPOOL_TIMEOUT = 120
 # Sentinel state used when we fail to query the application for its state
 NON_RESPONSIVE_STATE = 'NON_RESPONSIVE'
+# Maximum number of YARN high-availability (HA) redirects to follow before
+# giving up.
```

Comment:
can you elaborate a bit about what these redirects are

Reply:
Will add more detail to the comment.